### PR TITLE
Initial commit of text

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

File: adding-more-native-to-your-react-native-app.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# React Native Link
+
+### Who am I?
+
+- Eloy Durán
+- @alloy
+- Author of CocoaPods
+- Engineer at Artsy, where we’ve been using React Native for 3.5 years in our brownfield app
```
Comment (alloy) [inline adding-more-native-to-your-react-native-app.md]: @bartolkaruza We should also include your pluggables somewhere. Perhaps on the final slide that we can leave up during QA? What details do you want to include?

File: adding-more-native-to-your-react-native-app.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+1. The user adds a reference to the package’s Xcode project to their own (app) Xcode project;
+1. after which they are able to select the package’s static library from the list of libraries their application should
+   link against;
+1. and finally, where necessary, update build settings of the app Xcode project required by the dependency, such as
+   including the correct C++ standard library.
+1. If, at runtime, you are only using the package through JavaScript then you are done now. If you also need to use the
+   package’s code from your own native code, you will need to setup header search paths so you can include the package’s
+   headers in your source files.
+1. If the package has any native dependencies of its own, you need to repeat the above steps for those dependencies.
```
Comment (bartolkaruza) [inline adding-more-native-to-your-react-native-app.md]: would be nice to show a diff of project.pbxproj after this step. 

Later show the diff (or lack thereof) of the new approach. That's a big contrast between manual/react-native link and autolink

File: adding-more-native-to-your-react-native-app.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+native requirements and offloads the responsibility of configuring the app project to the dependency manager for the
+respective platform.
+
+For iOS, this means that adding a native package boils down to:
+
+1. Install the package, e.g. `yarn add lottie-react-native`.
+1. Run `pod install`.
+1. There’s no step 3.
+
+What this does under the hood, is that the CocoaPods manifest, the `Podfile`, includes a React Native specific helper,
```
Comment (bartolkaruza) [inline adding-more-native-to-your-react-native-app.md]: here maybe show the use_modules! line that needs to be added

File: adding-more-native-to-your-react-native-app.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+for Android apps, was already the default.)
+
+So, great–now there’s a single place where you manage your native dependencies! But given this standardization, we can
+take it one step further and automate the process. 🐢s all the way down!
+
+### Auto-linking
+
+Enter auto-linking, which is a new addition to the `react-native` command-line tool that discovers packages that have
+native requirements and offloads the responsibility of configuring the app project to the dependency manager for the
+respective platform.
```
Comment (alloy) [inline adding-more-native-to-your-react-native-app.md]: @bartolkaruza You mentioned that the Android side of auto-linking does not yet support transitive dependencies. Should I mention that hereabout? And can you give me a summary of why it doesn’t do that yet? (I assumed that through Gradle that would be handled transparently.)

Comment (alloy) [reply adding-more-native-to-your-react-native-app.md]: Hmm, unsure what you mean here, let’s catchup IRL at the meetup.

**Pull request merged**